server/controllers/RequestsController.js

```diff
@@ -22,6 +22,33 @@ class RequestController extends Controller {
       requests,
     });
   }
+
+  /**
+   * @description Fetch a request of a logged in user
+   *
+   * @param {Object} req - HTTP Request
+   * @param {Object} res - HTTP Response
+   *
+   * @return {Object} Returned object
+   */
+  static getRequest(req, res) {
+    const requestDetails = requests.find(request =>
```

Comment:
Again you have to cover edge cases. Do you know what happens if I try to a parsInt() on a string?

Reply:
There is a validation middleware that checks the id to make sure it is a number. it is in the validation folder